<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RunExistingMacro</class>
 <widget class="QMainWindow" name="RunExistingMacro">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>931</width>
    <height>447</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="RunMacroSlot1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>361</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string> Unnamed Macro 1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RunMacroSlot2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>361</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string> Unnamed Macro 2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RunMacroSlot3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>361</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string> Unnamed Macro 3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RunMacroSlot4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>361</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string> Unnamed Macro 4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RunMacroSlot5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>361</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string> Unnamed Macro 5</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="UserLogArea">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>100</y>
      <width>261</width>
      <height>281</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>259</width>
       <height>279</height>
      </rect>
     </property>
     <widget class="QScrollBar" name="UserLogScrollBar">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>0</y>
        <width>20</width>
        <height>281</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="RunButton">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>150</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>run now</string>
    </property>
   </widget>
   <widget class="QLabel" name="MacroSelectedLabelTitle">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>20</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Macro Selected:</string>
    </property>
   </widget>
   <widget class="QLabel" name="MacroSelectedLabel">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>20</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Selected Macro</string>
    </property>
   </widget>
   <widget class="QLabel" name="runhowmanytimeslabel">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>10</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>run how many times?: </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="timesToRun">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>20</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="scheduleButton">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>100</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>schedule run</string>
    </property>
   </widget>
   <widget class="QPushButton" name="MenuReturn2">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>320</y>
      <width>251</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Return To Main Menu</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="runUntilStoppedBox">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>50</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Run until stopped</string>
    </property>
   </widget>
   <widget class="QDial" name="speedDial">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>210</y>
      <width>50</width>
      <height>64</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>270</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Speed: </string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>931</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
